docs/api/1.0/index.rst

```diff
@@ -0,0 +1,96 @@
+.. _kinto-api-endpoints:
```

Comment:
nit: If we plan on having `.. added-on:: 1.X` in the doc, then the folder could be named `1.X` instead of `1.0` :)